__tests__/app.test.js

```diff
@@ -4,6 +4,7 @@ const testData = require("../db/data/test-data");
 const seed = require("../db/seeds/seed");
 const db = require("../db/connection");
 const endpointsJson = require("../endpoints.json");
+const { string } = require("pg-format");
```

Comment:
usually we'd import this as:
`const format = require("pg-format");`
To keep it semantically accurate

Also, do you need this at this point? not seen a use of pg-format in this PR. If you're not including it yet, then there's no need to require it in. Try keep PRs strictly what's relevant to the current task!

Reply:
that's an auto-import i haven't caught, i've not tried to bring that in myself lol dw. I'll get rid of it